src/doc/guide.md

```diff
@@ -599,7 +590,7 @@ let y = if x == 5i {
 }; // y: int
 ```
 
-Which we can (and probably should) write like this:
+We can also write it like this, which might make things more clear:
```

Comment:
I'm not sure I like my change now, but I don't like the original either:

Rust is opinionated, so the guide should be too, right?  How about:
"We can and should write it like this:"